mosviz/plugins/cutout_tool.py

```diff
@@ -136,11 +136,17 @@ def collect_text(self):
             self.save_path  = self.spec_path  
 
         if self.xSize.text() != "":
-            self.cutout_x_size = float(self.xSize.text())
+            try:
+                self.cutout_x_size = float(self.xSize.text())
+            except:
```

Comment:
You should make the exception explicit (I'm assuming in this case you're expecting that the string might be a float, and will fail with a `ValueError` exception) or use a `QValidator`.